arboretum/permissions/templates/reports/permissions/org_collaborators.md.tmpl

```diff
@@ -0,0 +1,50 @@
+{#- -*- mode:jinja2; coding: utf-8 -*- -#}
+# {{ test.title }} for {{ now.strftime('%Y-%m-%d') }}
+
+This report displays direct collaborators in the repositories of the organizations.
+
+It issues failures when:
+
+- Users are found as direct collaborators in any repository of the organizations
+
+It issues warnings when:
+
+- Users are found as direct collaborators in any repository of the organizations
+but they are allowed as execptions
+
+If a failure or warning is not present for the above category of check this
+means that the check passed without issue.
```

Comment:
I think to support and make the above statement more clear, you should provide a list of orgs and optionally the repos in those orgs that this check is working on.  This seems pretty important for this check given that if your config does not include any `direct` collaborator_types then the check is essentially a no-op.

Maybe up at the top where we say `This report flags direct collaborator access found in the organization repositories.` you provide a list of the orgs (and repos if applicable) that this check/report is reporting for.

You have access to the `test` object in the report template which has the `config` on it.